Corona-Warn-App/src/main/res/values-en/strings.xml

```diff
@@ -693,7 +693,7 @@
     <!-- YTXT: subtitle for legal information page, contact section -->
     <string name="information_legal_subtitle_contact">"E-mail: CoronaWarnApp@rki.de"</string>
     <!-- YTXT: subtitle for legal information page, open contact form -->
-    <string name="information_legal_subtitle_contact_form"><a href="https://www.rki.de/SharedDocs/Kontaktformulare/weitere/Corona-Warn-App/Corona-Warn-App_Integrator.html">"Contact Form"</a></string>
+    <string name="information_legal_subtitle_contact_form"><a href="https://www.rki.de/SharedDocs/Kontaktformulare/en/Kontaktformulare/weitere/Corona-Warn-App/Corona-Warn-App_Integrator.html">"Contact Form"</a></string>
```

Comment:
As we are currently discussing how we can provide this link from our translation system I would like to keep this PR open as the bug wouldn't be solved by only adding this fix.